dags/ods/kktix_ticket_orders/udfs/gather_town_loader.py

```diff
@@ -0,0 +1,38 @@
+"""
+load user's name, email etc into gather town whitelist
+please refer to this document for details: https://hackmd.io/PM_sWO5USo6dxMqT1uCrCQ?view
+"""
+import requests
+import tenacity
+from airflow.hooks.http_hook import HttpHook
+from airflow.models import Variable
+
+RETRY_ARGS = dict(
+    wait=tenacity.wait_none(),
+    stop=tenacity.stop_after_attempt(3),
+    retry=tenacity.retry_if_exception_type(requests.exceptions.ConnectionError),
+)
+
+GATHERTOWN_HTTP_HOOK = HttpHook(http_conn_id="gathertown_api", method="POST")
+
+
+def load(**context):
+    event_raw_data_array = context["ti"].xcom_pull(task_ids="GET_ATTENDEE_INFOS")
+    for event_raw_data in event_raw_data_array[:1]:
+        resp = GATHERTOWN_HTTP_HOOK.run_with_advanced_retry(
+            endpoint="/api/setEmailGuestlist",
```

Comment:
main logic is as follow:
1. get sensitive data like email and name from Airflow's xcom
2. invoke http post (provided by @GeekAngus 🙏 )